pkg/placement/monitoring/metrics.go

```diff
@@ -18,13 +18,31 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	activeActorTypesTotal = stats.Int64(
+		"placement/actortypes_total",
+		"The total number of active actor types reported to placement service.",
+		stats.UnitDimensionless)
+	activeNonActorTypesTotal = stats.Int64(
+		"placement/nonactortypes_total",
+		"The total number of active non actor types reported to placement service.",

```

Comment:
This should changed to The total number of non-actor hosts reported to placement service.